docs/modding/suitmodding/addinglocals.md

```diff
@@ -0,0 +1,186 @@
+2. In the manifest, add the following block of code to create a **localisation override**:
+   ```json
+   "localisationOverrides": {}
+   ```
+   Hit **Enter** to keep your code readable.
+
+3. **Insert the LOCR file hash** from the file you opened earlier. It will look something like this:
+   ```json
+   "localisationOverrides": {
+       "00985A1100E5EDDC": {}
+   }
+   ```
+
+4. Now, specify the **language** you want to override (e.g., `"english"`). Inside the language block, insert your **decimal number** and the **new suit name** in quotation marks:
+   ```json
+   "localisationOverrides": {
+       "00985A1100E5EDDC": {
+           "english": {
+               "1047194709": "New Suit Name"
+           }
+       }
+   }
+   ```
+
+5. If you also want to change the **suit description**, repeat the process by adding the decimal and the new text for the description.
+
+Here’s an example of both a name and description override:
+```json
+"localisationOverrides": {
+    "00985A1100E5EDDC": {
+        "english": {
+            "1047194709": "Blood Money Suit with Drip",
+            "3718104824": "The Blood Money suit but with extra style."
+        }
+    }
+}
+```
+
+---
+
+## Step 5: Add Multiple Languages or LOCR Files (Optional)
+
+If you want to add overrides for multiple languages, follow the same structure. For example, here’s how you can add French translations:
+
+```json
+"localisationOverrides": {
+    "00985A1100E5EDDC": {
+        "english": {
+            "1047194709": "Blood Money Suit with Drip",
+            "3718104824": "The Blood Money suit but with extra style."
+        },
+        "french": {
+            "1047194709": "Costume d'argent du sang avec goutte à goutte",
+            "3718104824": "Le costume de l'argent du sang mais avec un style supplémentaire."
+        }
+    }
+}
+```
+
+### Adding Another LOCR File
+
+If you want to modify another LOCR file, you can add additional entries in the `"localisationOverrides"` section. Simply separate them with a comma. Here’s an example of adding another LOCR file:
+
+```json
+"localisationOverrides": {
+    "00985A1100E5EDDC": {
+        "english": {
+            "1047194709": "Blood Money Suit with Drip",
+            "3718104824": "The Blood Money suit but with extra style."
+        }
+    },
+    "004B8C5124A49543": {
+        "english": {
+            "987654321": "Another Suit Name"
+        }
+    }
+}
+```
+
+---
+
+## Step 6: Save and Deploy
+
+After you’ve made all your changes, save the manifest file and deploy the mod as you normally would. Your suit names and descriptions will now be updated in-game.
+
+---
+
+With these steps, you should be able to modify suit names, descriptions, and other localisation text easily. Don’t hesitate to experiment with other text in the game once you’re comfortable!
+
+---
+
+This version is more structured, gradually introducing the steps with clear explanations. Let me know if this feels clearer and easier to follow!
```

Comment:
Did ChatGPT write that part or

Reply:
Yes. I wrote the article myself but the markdown syntaxing was done with gpt.